pkgs/top-level/python-packages.nix

```diff
@@ -7405,6 +7405,25 @@ in {
 
   rxv     = callPackage ../development/python-modules/rxv     { };
 
+  typechecks = callPackage ../development/python-modules/typechecks { };
+
+  datacache = callPackage ../development/python-modules/datacache { };
+
+  memoized-property = callPackage ../development/python-modules/memoized-property { };
+
+  gtfparse = callPackage ../development/python-modules/gtfparse { };
+
+  serializable = callPackage ../development/python-modules/serializable { };
+
+  tinytimer = callPackage ../development/python-modules/tinytimer { };
+
+  pyensembl = callPackage ../development/python-modules/pyensembl { };
+
+  gffutils = callPackage ../development/python-modules/gffutils { };
+
+  scikit-plot = callPackage ../development/python-modules/scikit-plot { };
+
+  pybedtools = callPackage ../development/python-modules/pybedtools { };
```

Comment:
please attempt to sort these

Reply:
it has been evolving over time... some people grouped similar packages, some people just dropped them anywhere. At the very least, random listings should be avoided